api/models/prediction_job.go

```diff
@@ -45,6 +45,7 @@ type Config struct {
 	ServiceAccountName string                        `json:"service_account_name"`
 	JobConfig          *spec.PredictionJob           `json:"job_config"`
 	ImageRef           string                        `json:"image_ref"`
+	MainAppPath        string                        `json:"main_app_path"`
```

Comment:
Will this config be stored in DB

Reply:
Nope. It won't be. This is related to db?

Reply:
Ah it's stored in db but doesn't require new column